buildingmotif/api/views/model.py

```diff
@@ -200,9 +204,17 @@ def validate_model(models_id: int) -> flask.Response:
                 "message": f"Libraries with ids {nonexistent_libraries} do not exist"
             }, status.HTTP_400_BAD_REQUEST
 
-    # if shape_collections is empty, model.validate will default
-    # to the model's manifest
-    vaildation_context = model.validate(shape_collections)
+    # temporarily change the SHACL engine
+    bm = get_building_motif()
```

Comment:
This isn't needed anymore with the change you made on the last PR, right?

Reply:
Correct, but that will be fixed in the other PR I believe